packages/ocap-kernel/src/store/methods/object.ts

```
@@ -31,6 +33,7 @@ export function getObjectMethods(ctx: StoreContext) {
     const koId = getNextObjectId();
     ctx.kv.set(getOwnerKey(koId), owner);
     setObjectRefCount(koId, { reachable: 1, recognizable: 1 });
+    setRevoked(koId, false);
```

I think that revocation is likely to be _extremely_ rare, as it's violent and disruptive in ways that make defensive programming harder. In many years of using ocap patterns, I've never encountered a need or even a real desire for it. The availability of a revocation mechanism seems to mostly be used as an intellectual bridge to reassure people who are new to the ocap world that things they are worried about will be OK. Given that rarity, I'm unconvinced that affirmatively marking every unrevoked object as unrevoked pays for itself.  I think it's probably better to have the simple lack of a revoked flag serve as the indicator that the object is unrevoked (i.e., in the absence of any information to the contrary, the object still being active is the default).  I'm also concerned about the performance impact of an additional database read on every object reference lookup -- actually, I'm not 100% convinced that explicitly marking the revocation status at all is the right thing, since we already have to deal with the case of the kref being absent anyway. The main benefit I see of having the revoked flag is that we can give explicit feedback that the reason an operation failed was revocation (and, in line with my earlier comment about revocation being rare, it's almost certainly not going to be storage space issue to hold the `true` revoked flags in this case), but I'm not sure that giving explicit "this has been revoked" feedback to callers is what we want from a security perspective as it's a way (albeit a weak one) for an outsider to probe the vat's internal state. In other words, we might WANT it to be impossible for an outsider to distinguish between a reference to a revoked object and a reference that is just bad.

---

On a first reading I agree with @FUDCo that the absence of the `revoked` flag to `true` is enough to signify that it isn't revoked.

Also would it be better to throw an unreachable error instead of returning a body with `revoked object`?